### PR TITLE
Wrong command not existing on windows 10

### PR DESCRIPTION
**Pull request by s-krawczyk**

### CODE REVIEW

File: virtualization/windowscontainers/quick_start/quick_start_windows_10.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ This quick start is specific to Hyper-V containers on Windows 10. Additional qui
 The container feature needs to be enabled before working with Windows containers. To do so run the following command in an elevated PowerShell session.
 
 ```none
-Enable-WindowsOptionalFeature -Online -FeatureName containers -All
```
Comment (taylorb-microsoft) [inline virtualization/windowscontainers/quick_start/quick_start_windows_10.md]: This is the correct command in an elevated PowerShell window.

**Comment by taylorb-microsoft** [comment]
In a PowerShell window Enable-WindowsOptionalFeature is the correct command.  Do we need to call out PowerShell more explicitly?

**Comment by rs38** [comment]
for me it wasn't obvious that this powershell command even exists. good to know

**Comment by s-krawczyk** [comment]
My bad. I looked into Server 2016 configuration file.

Pozdrawiam
SK

2016-10-05 12:21 GMT+02:00 rs38 <notifications@github.com>:

**Comment by scooley** [comment]
I'll add a note about making sure you're in an elevated session if the command doesn't exist.

With that caveat, can I close this PR?

**Comment by s-krawczyk** [comment]
Yes, thx.

Pozdrawiam
SK

2016-10-25 20:36 GMT+02:00 Sarah Cooley <notifications@github.com>:

**Comment by scooley** [comment]
Done :)

**Pull request closed without merging**